react/javascript/src/components/gherkin/MDG.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import rehypePlugins from '../app/rehypePlugins'
+import GherkinQueryContext from '../../GherkinQueryContext'
+import Step from './Step'
+import styles from '../app/GherkinDocumentList.module.scss'
+
+type IProps = { uri: string; children: any }
+
+const MDG: React.FunctionComponent<IProps> = ({ uri, children }) => {
+  const gherkinQuery = React.useContext(GherkinQueryContext)
+
+  return (
+    <div className={styles.list}>
+      <ReactMarkdown
+        rehypePlugins={rehypePlugins}
+        components={{
+          ul({ children }) {
+            return <ul className={'cucumber-steps'}>{children}</ul>
```

Comment:
I had to add the `cucumber-steps` class here in order to get the `list-style: none` CSS rule, but the drawback is that *any* `<li>` now gets that style, including list items that are *not* Gherkin steps.

I wonder if it would be better to move this style up to the dom element that renders the enclosing `Scenario`. That way the style would only apply to Gherkin Step list items.

Reply:
Addressed by changing to

```scss
.cucumber-steps {
  padding: 0;
  margin: 1em;

  > li {
    list-style: none;
  }
}
```

Reply:
Do we support using ordered lists as well with MDG?

Reply:
You mean this?

```markdown
1. one
1. two
1. three
```

Which renders as:

1. one
1. two
1. three

MDG doesn't currently support it, but it definitely should! I'll add support for that

Reply:
Pushed something that deals with lists better - when we come to render a `ul`, we check if there's a step on that line (since the `ul` and its first `li` will have the same starting line in the source) - if so we render a steps list, otherwise just a regular one (with bullets).